Automattic/wp-calypso
# Earn: Add a Stripe connect prompt to the Donations block.

This PR adds a Stripe connection prompt to the (in development) Donations block, for sites that have the appropriate plan but have not yet set themselves up with Stripe.

#### Testing instructions

* On this branch locally, run `yarn dev --sync` to sync the changes to your sandbox.
* On your sandbox, fetch the new `wpcom-block-editor` build generated on this PR: `./bin/calypso/build-wpcom-block-editor.sh 770978`.
* Sandbox the API, `widgets.wp.com`, the store, and a test site that has at least a Personal plan, but no Stripe account connected.
* On that test site, start a new post at `/wp-admin/post-new.php`.
* Enable console monitoring of our Tracks events with `localStorage.setItem( 'debug', 'wpcom-block-editor:*' );`.
* Add a Donations block.
* Verify the Donations block loads with the three tabbed view visible and working, with the prompt to connect to Stripe above it.
* Click the "connect" button and verify you are sent to a Stripe connect form. With the store sandboxed, you should be able to click the top banner to connect in development mode.
* Finish connecting, and verify the connect prompt is no longer there when you go back and refresh the editor.
* Verify a Tracks event in the console for `wpcom_block_donations_stripe_connect_click `.
